db/schema.rb

```diff
@@ -103,7 +103,6 @@
     t.index ["reset_password_token"], name: "index_users_on_reset_password_token", unique: true
   end
 
-  add_foreign_key "responses", "emotions"
```

Comment:
@VitaliiShevchenko sorry, but I don't understand why you remove this `foreign_key`? I don't see any code that could cause this.

Reply:
@SerhiiBorozenets  This is an auto-generated comment dependent on migrations that are not related to this PR

Reply:
@VitaliiShevchenko you should run migration locally in this PR and update PR

Reply:
![image](https://user-images.githubusercontent.com/59101468/232464405-0f6562ce-c2f7-4d37-baa4-7f3798024e24.png)

Reply:
@VitaliiShevchenko I pulled this PR, run migration and I have changes. Try drop your db and run migration again.